validator-api/src/rewarding/mod.rs

```diff
@@ -45,8 +44,6 @@ pub(crate) const REWARDING_GAS_LIMIT_MULTIPLIER: f64 = 1.05;
 
 pub(crate) const MAX_TO_REWARD_AT_ONCE: usize = 50;
 
-pub(crate) const REWARDING_TIME_VARIANCE: f32 = 0.05; // 5% (so for example +/-1.2h for 24h epoch)
```

Comment:
Why not just put a very small value here (not sure if 0 works, but something close to that)? That way we have an easy way of re-enabling it in the future?

Reply:
Because in hindsight I'm not entirely sure whether it was a good idea to begin with. It could bring more harm than benefit by being nondeterministic. And the ranges of values change drastically if we go from 1 day rewarding interval into 1 month. I think we should try the problem in a different way once we get to that point.